exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -113,6 +116,16 @@ def _get_resource_data(
     return resource_data
 
 
+def _load_credential_from_file(filepath) -> ChannelCredentials:
+    try:
+        with open(filepath, "rb") as f:
```

Comment:
Tried to use the OTLP Exporter today and I think I ran into an issue. If you don't have the `OTEL_PYTHON_EXPORTER_OTLP_CERTIFICATE` Env variable set, then the call to `Configuration().EXPORTER_OTLP_CERTIFICATE` 

https://github.com/jan25/opentelemetry-python/blob/2ff2492333ab85d10b81a89f86d48a6eaad74912/exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py#L175

will return `None` instead of a `String`. Then this line will throw error 

`TypeError: expected str, bytes or os.PathLike object, not NoneType`

which is not caught by `FileNotFoundError` below.

Reply:
I suggest opening a new issue to track these problems instead of continuing an already closed PR thread.